### ISSUE TITLE
feature/view patient's encounter diagnoses in service request

### ISSUE THREAD

**Issue by chris-dare** [serenity-health/roadmap#29]
**Tell us about your request!**
View patient's visit diagnoses on service request. In addition to that, biomedical scientists should be able to see the patient's encounter history in the diagnostic workspace

**Which app does this concern?**
Provider portal

**What will be benefit of this feature?**
Viewing this information helps a biomedical scientist validate their test results

**Describe the solution you'd like**
- The service request detail view will dispay the patient's entire encounter diagnoses from their visit
- The patient profile in the diagnostic workspace will also have the patient's encounter history

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by chris-dare** [comment]
@TuyizereBapt please create a field on ServiceRequest called `encounter_diagnoses`.
Whenever a service request is created, we should grab the names of all diagnoses in the encounter and place then in a string joined by `" | "` in that field in ServiceRequest

**Comment by WealthSecrets** [comment]
@TuyizereBapt

**Comment by TuyizereBapt** [comment]
> of all diagnoses in the encounter

Why can't this instead be a property on the ServiceRequest? I think with this approach, you can even see Diagnoses carried out after the ServiceRequest was created. Is it supposed to be Diagnoses up until the ServiceRequest is created?

**Comment by chris-dare** [comment]
> > of all diagnoses in the encounter
> 
> Why can't this instead be a property on the ServiceRequest? I think with this approach, you can even see Diagnoses carried out after the ServiceRequest was created. Is it supposed to be Diagnoses up until the ServiceRequest is created?

Yes please create it as a field on the ServiceRequest. At the point of creation, it should just take all the encounter diagnoses recorded in that encounter and save it in that field. 
It may be helpful to actually implement this as a ListField
